.github/workflows/hide_bot_comments.yml

```diff
@@ -0,0 +1,24 @@
+name: Hide Bot Comments
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
+
+jobs:
+  hide-bot-comments:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: Install Requests
+        run: |
+          pip install requests
+      - name: Hide Bot Comments
+        run: |
+          python .github/workflows/main.py
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The available per default `secrets.GITHUB_TOKEN` only gives read-only permissions IIRC

Reply:
how does this one work then? https://github.com/conan-io/conan-center-index/pull/4876/files#diff-9364034bde9293a8a9265833fd8049a74b9b0987bb9bf5257cf02b39f49a9c77R16
it adds labels, which should be a write action, isn't it?
@prince-chrismc do you have an idea?

Reply:
yes, I wonder, as if we have actions which add labels and create PRs, it should be fine...

Reply:
Weird. I'm pretty sure I had cases in the past where I was forced to create manually an access token because the default one was not enough for the own (non-fork) repository 🤔

Reply:
Not sure if [Actions Permissions](https://docs.github.com/en/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository#allowing-specific-actions-to-run) can disable it.

Reply:
I had to create manually tokens, when I needed to write to other repositories, but it should befine using GITHUB_TOKEN as long as you want to write to the same repository the action lives in.

Reply:
On mobile but if I understand correctly, this action is running as the default GA token to edit a _different user_ @conan-center-bot's comments.

I suspect that level of authority is not provided the default has Read/write to the repo but that's still very limited in terms of managing contributions from others.

@ericLemanissier comment above has a good link.

Reply:
I did some research on this https://github.com/prince-chrismc/label-merge-conflicts-action#faq---how-do-i-fix-resource-not-accessible-by-integration

I can confirm the comments above

You'll need to use a PAT